app/controllers/AnnotationController.scala

```diff
@@ -436,7 +436,7 @@ class AnnotationController @Inject()(
     sil.SecuredAction.async { implicit request =>
       for {
         annotation <- provider.provideAnnotation(typ, id, request.identity)
-        _ <- bool2Fox(annotation._user == request.identity._id && annotation.visibility != AnnotationVisibility.Private) ?~> "notAllowed" ~> FORBIDDEN
+        _ <- bool2Fox(annotation._user == request.identity._id) ?~> "notAllowed" ~> FORBIDDEN
```

Comment:
I removed the `&& annotation.visibility != AnnotationVisibility.Private` bit, since I found it weird that I could not change the may-others-edit boolean (even if it was true), if the visibility was set to true. also the user might want to enable its team manager to edit the annotation, for example.